website/server/controllers/api-v3/user.js

```diff
@@ -408,11 +408,13 @@ const partyMembersFields = 'profile.name stats achievements items.special';
  * @apiGroup User
  *
  * @apiParam {String=fireball, mpHeal, earth, frost, smash, defensiveStance, valorousPresence, intimidate, pickPocket, backStab, toolsOfTrade, stealth, heal, protectAura, brightness, healAll} spellId The skill to cast.
- * @apiParam (Body) {UUID} targetId Query parameter, necessary if the spell is cast on a party member or task. Not used if the spell is case on onesself or the user's current party.
+ * @apiParam (Query) {UUID} targetId Query parameter, necessary if the spell is cast on a party member or task. Not used if the spell is case on onesself or the user's current party.
```

Comment:
@taldin The test failure might be just a timeout. Can you please change this:
"Query parameter, necessary if the spell is cast on a party member or task. Not used if the spell is case on onesself or the user's current party."
to this:
"Query parameter, necessary if the skill is cast on a party member or task. Not used if the skill is cast on the user or the user's current party."
which will fix a couple of problems with the text and will also restart the tests. :)